docs/team.rst

```diff
@@ -43,278 +34,275 @@ How teams operate and interact
+    - Maintain documentation on starting meetups
 
-Membership
-  - The conference chair appoints team members for each conference
-  - Conference-specific roles are defined in `Event Roles <https://www.writethedocs.org/organizer-guide/confs/event-roles/>`__ and are subject to change based on regional requirements
+  Membership
+    - Previous meetup organization experience preferred but not mandatory
 
-Members (updated to reflect latest year)
-  * :doc:`Portland 2021 Team </conf/portland/2021/team>`
-  * :doc:`Prague 2021 Team </conf/prague/2021/team>`
-  * :doc:`Australia 2021 Team </conf/australia/2021/team>`
+//TODO choose team lead for community board
 
-.. _moderation-team:
+  Members
+    - `Rose Williams <https://twitter.com/ZelWms>`_ 
+    - `Alyssa Whipple Rock <https://alyssarock.pro/>`_
 
-Slack Moderation
-................
 
-Responsibilities
-  - Monitor the WTD Slack space and handle CoC issues, rotation 
-  - Full team procedures and processes are currently in the process of being finalized in a separate WEP: WEP0002
-    
-Membership
-  - Team members must have some previous experience with CoC handling or community moderation
-  - Team members must be active in the WTD Slack space and available for moderation shifts
+.. _newsletter-team:
 
-Members
-  * `Janine Chan <https://www.linkedin.com/in/janinechan/>`_ ``@janine`` *
-  * `Daniel Beck <https://twitter.com/ddbeck>`_ ``@ddbeck``
-  * `Samuel Wright <https://twitter.com/plaindocs>`_ ``@plaindocs``
-  * `Ravind Kumar <https://www.linkedin.com/in/ravind-kumar-b4813650/>`_ ``@ravindk-minio``
-  * `Eric Holscher <https://twitter.com/ericholscher>`_ ``@ericholscher``
+Newsletter
+..........
 
+  Responsibilities
+    - Curate Slack conversations and distill them into brief newsletter stories
+    - Write a monthly update on general goings-on in the community
+    - Round up any WTD events or meetups for the upcoming month
+    - Assemble and publish the monthly newsletter to the mailing list and the website
 
-Meetups
-.......
+  Membership
+    - Team members must have English editorial skills
 
-Responsibilities
-  - Support local and regional meetups 
-  - Create and manage meetup.com accounts
-  - Assist with CoC escalations from meetup organizers
-  - Maintain a mailing list for meetup organizers
-  - Update website with new meetups
-  - Socialize information around meetups (new meetup topics, speaker ideas)
-  - Maintain documentation on starting meetups
+  Members
+    * `Aaron Collier <https://github.com/CollierCZ>`_ (team lead)
+    * `Hillary Fraley <https://github.com/hillaryfraley>`_
+    * `Claire Lundeby <https://twitter.com/clairelundeby>`_
+    * Royce Cook
+    * `Heather Zoppetti <https://github.com/hzoppetti>`__
 
-Membership
-  - Previous meetup organization experience preferred but not mandatory
 
-Members
-  * `Rose Williams <https://twitter.com/ZelWms>`_ *
-  * `Alyssa Whipple Rock <https://alyssarock.pro/>`_
```

Comment:
Ouch! Rough way to find out I'm being demoted, I guess. 😅 

Anything I can do to get back into your good graces?

Reply:
Oh no no no @barbaricyawps have no fear, this part of the page hasn't been updated since we originally published the team list, and I missed it when originally preparing the PR. This is why these feedback rounds are so important, thanks for a most important catch!

Reply:
That's a relief! I just wasn't sure what this meant. 

Let me know if you need me to clarify what the Write the Docs Quorum program is and how it fits into the org overall. I'm happy to fit that program in wherever it makes sense in the org to place it. 😄

Reply:
Hey @thatdocslady Likewise, I will also want to know how the Write the Docs Africa  could fit into the org

Reply:
@Dule-martins I think that y'all are currently in the [meetup section](https://www.writethedocs.org/meetups/#current-meetups-africa) of the site. If you have aspirations to do more things though we can definitely discuss what that might look like!